### PR TITLE
CONTROLLER: Add Post Refugee

### LINKED ISSUE #10: CONTROLLER: POST Refugee

### PR DESCRIPTION
**Pull request by Spectralgo**
Closes #10

### CODE REVIEW

File: RefugeeLand.Core.Api/Controllers/RefugeesController.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+[ApiController]
+[Route("api/[controller]")]
+public class RefugeesController : RESTFulController
+{
+    private readonly IRefugeeService refugeeService;
+
+    public RefugeesController(IRefugeeService refugeeService)
+    {
+        this.refugeeService = refugeeService;
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Controllers/RefugeesController.cs:18]: Make it fat arrowed method

File: RefugeeLand.Core.Api/Controllers/RefugeesController.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    [HttpPost]
+    public async ValueTask<ActionResult<Refugee>> PostRefugeeAsync(Refugee refugee)
+    {
+        throw new NotImplementedException();
+    }
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Controllers/RefugeesController.cs:25]: Make it fat arrowed method

File: RefugeeLand.Core.Api/Controllers/RefugeesController.cs (hunk 1 above)
Comment (Spectralgo) [reply RefugeeLand.Core.Api/Controllers/RefugeesController.cs]: Thx @ElbekDeveloper, I'll try to be more vigilant :)

Comment (Spectralgo) [reply RefugeeLand.Core.Api/Controllers/RefugeesController.cs]: Done

File: RefugeeLand.Core.Api/Controllers/RefugeesController.cs (hunk 2 above)
Comment (Spectralgo) [reply RefugeeLand.Core.Api/Controllers/RefugeesController.cs]: done

File: RefugeeLand.Core.Api/Controllers/RefugeesController.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using System;
```
Comment (ElbekDeveloper) [inline RefugeeLand.Core.Api/Controllers/RefugeesController.cs:1]: Header

**Pull request closed without merging**